examples/apps/data-object-grid/tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.base.json",
 	"compilerOptions": {
-		"module": "esnext",
-		"target": "es6",
-		"outDir": "lib",
-		"jsx": "react",
+		"module": "ESNext",
+		"moduleResolution": "Bundler",
+		"outDir": "./lib",
```

Comment:
```suggestion
		// ESNext/Bundler works better than Node16/Node16 for this example because of the <Collapsible> React control
		// and it is intended to be used via a bundler anyway
		"module": "ESNext",
		"moduleResolution": "Bundler",
		"outDir": "./lib",
```